### DISCUSSION TITLE
Display Geometry as formatted JSON

### DISCUSSION THREAD

**Discussion by vidhav** [directus/directus#7879] [Feature Requests]
It seems like the geometry type returns [GeoJSON](https://geojson.org/) (which is just standardised JSON) when displayed in a Layout. The only option now is "Raw Value", but it could be possible to use the "Formatted JSON value" display to extract information.

Example value for Point:
`{"coordinates": [5.9974, 58.4625], "type": "Point"}`

Display Template:
`{{ coordinates[0] }}° east, {{ coordinates[1] }}° north`

**Comment by rijkvanzanten** [comment]
Wanna open a PR for that?? Should be as simple as adding geometry to the supported types here:

https://github.com/directus/directus/blob/main/app/src/displays/formatted-json-value/index.ts#L8

**Reply by vidhav** [reply]
Sure! I'll do that!

**Comment by br41nslug** [comment]
has been implemented
